abmarl/policies/q_table_policy.py

```diff
@@ -0,0 +1,156 @@
+
+from abc import ABC, abstractmethod
+
+from gym.spaces import Discrete
+import numpy as np
+
+from .policy import Policy
+
+
+# TODO: Remove this class and let the subclasses, like greedy policy, work with
+# different implementations of the Q insead of forcing it to be a table.
+class QTablePolicy(Policy, ABC):
+    """
+    A policy that explicitly stores and updates a Q-table.
+
+    This requires Discrete observation space and action space.
+    """
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Since we overwrite the properties anyways, what if we put the obs and action space as inits here as well and forget about the `super()`?